crates/biome_ungrammar/src/lexer.rs

```diff
@@ -161,5 +161,5 @@ fn is_whitespace(c: char) -> bool {
     matches!(c, ' ' | '\t' | '\n')
 }
 fn is_ident_char(c: char) -> bool {
-    matches!(c, 'a'..='z' | 'A'..='Z' | '_')
+    matches!(c, 'a'..='z' | 'A'..='Z' | '_' | '0'..='9')
```

Comment:
Could you revert this change since you don't need it anymore?

Reply:
Thanks for your comment. I corrected it